python/cudf/cudf/core/frame.py

```diff
@@ -917,6 +917,12 @@ def scatter_by_map(
         -------
         A list of cudf.DataFrame objects.
         """
+        if not isinstance(self, cudf.DataFrame):
+            warnings.warn(
+                f"{self.__class__}.scatter_by_map is deprecated and will be "
```

Comment:
This will read like: `<class 'cudf.core.index.Index'>.scatter_by_map is deprecated`. I think you want:
```suggestion
                f"{self.__class__.__name__}.scatter_by_map is deprecated and will be "
```

Reply:
Good call, yes.